lib/core/versions/latest/util/Compressor.ts

```diff
@@ -0,0 +1,60 @@
+import Encoder from '../Encoder';
+import util = require('util');
+import zlib = require('zlib');
+
+/**
+ * Encapsulates functionality to compress/decompress data.
+ */
+export default class Compressor {
+
+  private static readonly gzipAsync = util.promisify(zlib.gzip);
+  private static readonly gUnzipAsync = util.promisify(zlib.gunzip);
+
+  /**
+   * Compresses the data in gzip and returns it as base64url encoded string.
+   * @param input The buffer to compress.
+   */
+  public static async compressAsBase64Url (input: string): Promise<string> {

```

Comment:
Sanity: doesn't appear to be used anywhere except for test?

Reply:
Removing.